bears/configfiles/DockerfileLintBear.py

```diff
@@ -17,7 +17,7 @@ class DockerfileLintBear:
     more information .
     """
     LANGUAGES = {'Dockerfile'}
-    REQUIREMENTS = {NpmRequirement('dockerfile_lint', '0.2.7')}
+    REQUIREMENTS = {NpmRequirement('dockerfile_lint', '0')}
```

Comment:
no way; we do not support version 0-infinity.

I bet our tests would fail if someone tried to use most of the versions of this package.

Reply:
yes :P they'll fail for 0.2.7